e2e/Pages/DisasterTypeComponent.ts

```diff
@@ -1,22 +1,50 @@
 import { expect } from '@playwright/test';
+import { format } from 'date-fns';
 import { Locator, Page } from 'playwright';
 
 import DashboardPage from './DashboardPage';
 
 class DisasterTypeComponent extends DashboardPage {
   readonly page: Page;
   readonly topBar: Locator;
+  readonly timeline: Locator;
 
   constructor(page: Page) {
     super(page);
     this.page = page;
     this.topBar = this.page.getByTestId('dashboard-top-bar');
+    this.timeline = this.page.getByTestId('timeline-button');
   }
 
   async topBarComponentIsVisible() {
     const topBar = this.topBar;
     await expect(topBar).toBeVisible();
   }
+
+  async allDisasterTypeElementsArePresent() {
```

Comment:
This will fail for other countries where different hazards are enabled. This is **good for now** but good to know this when you run the same tests for a different login.